Jenkinsfile

```diff
@@ -147,9 +147,19 @@ pipeline {
                     }
                     post { always { retry(3) { deleteDir() } } }
                 }
+                stage('Unit with MPI') {
+                    agent any
+                    steps {
+                        unstash 'MathSetup'
+                        sh "echo CC=mpicxx >> make/local"
+                        sh "echo STAN_MPI:=true >> make/local"
```

Comment:
we can just leave this as `STAN_MPI=true` (to be consistent with everything else). There's no concern with this being reevaluated multiple times.

Reply:
So when do you use the `:=`? I thought it was whenever you didn't need lazy evaluation.